txscript/script_test.go

```diff
@@ -4301,6 +4301,28 @@ func TestIsUnspendable(t *testing.T) {
 				0xfa, 0x0b, 0x5c, 0x88, 0xac},
 			expected: false,
 		},
+		{
+			// Spendable
+			pkScript: []byte{0xa9, 0x14, 0x82, 0x1d, 0xba, 0x94, 0xbc, 0xfb,
```

Comment:
Whenever possible, it's nice to use human-friendly names for the opcodes. It's not necessary in this example, but I added some suggestions for the other test cases you added.